app/posts/apply-for-teacher-training/2023-10-31-blocking-candidates-with-no-visa.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Stopping candidates from applying for courses that do not sponsor visas
+description:
```

Comment:
need to add a description here. Maybe: We designed a pattern to stop international candidates applying for courses that cannot sponsor visas.

Reply:
Oh i thought I added, dont know where it went!